CMSIS/NN/Source/NNSupportFunctions/arm_nn_vec_mat_mult_t_s8.c

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (C) 2020 Arm Limited or its affiliates. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* ----------------------------------------------------------------------
+ * Project:      CMSIS NN Library
+ * Title:        arm_nn_vec_mat_mult_t_s8
+ * Description:  s8 vector by matrix (transposed) multiplication
+ *
+ * $Date:        30. January 2020
+ * $Revision:    V.1.0.0
+ *
+ * Target Processor:  Cortex-M
+ *
+ * -------------------------------------------------------------------- */
+
+#include "arm_math.h"
+#include "arm_nnfunctions.h"
+#include "arm_nnsupportfunctions.h"
+
+/**
+ * @ingroup groupSupport
+ */
+
+/**
+ * @addtogroup NNBasicMath
+ * @{
+ */
+
+/*
+   * s8 vector by matrix (transposed) multiplication
+   *
+   * Refer header file for details.
+   *
+   */
+arm_status arm_nn_vec_mat_mult_t_s8(const int8_t *lhs,
+                                    const int8_t *rhs,
+                                    const int32_t *bias,
+                                    int8_t *dst,
+                                    const int32_t lhs_offset,
+                                    const int32_t rhs_offset,
+                                    const int32_t dst_offset,
+                                    const int32_t dst_multiplier,
+                                    const int32_t dst_shift,
+                                    const int32_t k,
+                                    const int32_t n,
+                                    const int32_t activation_min,
+                                    const int32_t activation_max)
+{
+#if defined(ARM_MATH_DSP)
+    const int32_t off0 = k - 4;
+    const int16_t lhs_offset_s16 = lhs_offset;
+    const int16_t rhs_offset_s16 = rhs_offset;
+
+    const uint32_t lhs_offset_s16x2 = __PKHBT(lhs_offset_s16, lhs_offset_s16, 16);
+    const uint32_t rhs_offset_s16x2 = __PKHBT(rhs_offset_s16, rhs_offset_s16, 16);
+#endif
+
+    for(int32_t n_idx = 0; n_idx <= (n - 2); n_idx+=2)
```

Comment:
Please adjust the spacing between the variables (n_idx += 2) and in other places as well..